apps/backend/src/modules/groups/group.service.ts

```diff
@@ -46,8 +46,17 @@ class GroupService implements Service {
 		return item.toObject();
 	}
 
-	public delete(): ReturnType<Service["delete"]> {
-		return Promise.resolve(true);
+	public async delete(id: number): Promise<boolean> {
+		const isDeleted = await this.groupRepository.delete(id);
+
+		if (!isDeleted) {
+			throw new GroupError({
+				message: ExceptionMessage.GROUP_NOT_FOUND,
+				status: HTTPCode.NOT_FOUND,
+			});
+		}
+
+		return isDeleted;
```

Comment:
Maybe I should just return await this.groupRepository.delete(id); instead of this.

Reply:
I clarified this and you should keep this as it is. Check [this](https://bsa2024.slack.com/archives/C07HWM8616U/p1725378501043329).